source/core/src/main/com/csse3200/game/screens/AirAnimalSelectionScreen.java

```diff
@@ -0,0 +1,47 @@
+package com.csse3200.game.screens;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.ScreenAdapter;
```

Comment:
I'm going to assume just like the display this can probably be abstracted away into a SelectionScreen class & not have 3 duplicate files of code.